api/jobs/ping.js

```diff
@@ -4,7 +4,9 @@ const job = require('../parts/jobs/job.js'),
     log = require('../utils/log.js')('job:ping'),
     countlyConfig = require("../../frontend/express/config.js"),
     versionInfo = require('../../frontend/express/version.info'),
-    request = require('request');
+    request = require('request'),
+    plugins = require('../../plugins/pluginManager.js'),
+    offlineMode = plugins.getConfig("api").offline_mode;
```

Comment:
I think job is loaded on process start and does not reload.
So if you change config later, this would still be the same.

Also I think master process (which runs jobs) does not load configs at all, so you need to load them manually through `plugins.loadConfigs` inside the jobs run
https://github.com/Countly/countly-server/blob/master/plugins/pluginManager.js#L80

Reply:
same goes to other places too :)